tests/unit_tests/test_unit.py

```
@@ -674,13 +959,47 @@ def request(self, *args, **kwargs):
-            casanovo._get_model_weights(
-                selector, pathlib.Path(tmp_dir), (3, 0, 0)
+            shared_loading._get_model_weights(
+                selector, pathlib.Path(tmp_dir), (3, 0, 0), ckpt_model
             )
 
         assert mock_get.request_counter == 0
 
 
+def test_get_model_weights_no_github_checkpoints(monkeypatch, tmp_path):
+    class MockRepo:
+        def get_releases(self):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `get_releases`

(ANN202)